Editor/AGS.Editor/AGSEditor.cs

```diff
@@ -324,6 +326,28 @@ public void Dispose()
             _debugger.EditorShutdown();
         }
 
+        private void CreateAppSettings()
+        {
+            // Test for a valid config file, in case of corrupt config
+            // delete the file and notify the user that the prefs will be reset.
+            try
+            {
+                ConfigurationManager.OpenExeConfiguration(ConfigurationUserLevel.PerUserRoamingAndLocal);
+            }
+            catch (ConfigurationErrorsException ex)
+            {
+                string filename = ex.Filename;
+                MessageBox.Show("Editor's configuration is corrupt and cannot be loaded. " +

```

Comment:
Use `Factory.GUIController.ShowMessage` instead, it'll play nicer when we finally make improvements to editor running in headless mode as discussed in #666

Reply:
Fixed.